src/main/java/org/example/examplefinalProject/entity/Parent.java

```diff
@@ -20,8 +27,8 @@ public class Parent {
     @ManyToMany(fetch = FetchType.EAGER, cascade = {CascadeType.DETACH, CascadeType.MERGE, CascadeType.PERSIST, CascadeType.REFRESH})
     @JoinTable(
             name = "student_parent",
-            joinColumns = @JoinColumn(name = "parent_id"),
-            inverseJoinColumns = @JoinColumn(name = "student_id")
+            joinColumns = @JoinColumn(name = "parentId"),
+            inverseJoinColumns = @JoinColumn(name = "studentId")
```

Comment:
O par suborov nizsie to nastavujes presne naopak, preco unika mi nieco? Nerozumiem tomu.
![image](https://github.com/user-attachments/assets/c18c6dfd-9759-4e55-90ac-b8eb5eb716da)